### DISCUSSION TITLE
Register identity with user registration disabled

### DISCUSSION THREAD

**Discussion by foldess** [ory/.github#77] [Q&A]
As title states i want to know if there is a way to register new identities with user registration disabled. I want to be able to create new users from the dashboard but can't seem to find how to do it, do i have to temporarily enable user registration and disable it when i'm done?

**Comment by Benehiko** [comment]
Hi @foldess 

You should be able to create the identities through the admin API without needing to enable the self-service registration flow. https://www.ory.sh/docs/kratos/manage-identities/invite-users

You can also import identities into the system with their credentials. 
https://www.ory.sh/docs/kratos/manage-identities/import-user-accounts-identities
